### PR TITLE
Change rgl.* calls to *3d calls.

### PR DESCRIPTION
**Pull request by dmurdoch**
The surface3d() function needs an rgl fix that's in the about-to-be-submitted version 0.111.5, so I've added that version requirement.  Let me know if you'd prefer a run-time test and workaround for the old code.

### CODE REVIEW

**Comment by eddelbuettel** [comment]
The one thing I always ask about is to add an extry to ChangeLog.  Can you do that?  Standard old school GNU format with date, name, email separated by two spaces, bullets indented by eight or a tab.

**Comment by dmurdoch** [comment]
This commit should work with current CRAN rgl as well as the upcoming one.  After 0.111.5 is released, the version check code could be removed.

File: DESCRIPTION
Comment (eddelbuettel) [inline DESCRIPTION:3]: Please reverse this.  Release versions are for releases, this ain't one.

What I have been doing for years in all my packages is to add a .0 or .1 at the end and increment that. I consider this to be the task of the package maintainer and do not do it for them / impose it on them.

**Review by eddelbuettel — CHANGES_REQUESTED** [review]

File: DESCRIPTION
Comment (dmurdoch) [reply DESCRIPTION]: Done

**Comment by eddelbuettel** [comment]
Sorry for the hoops but I cannot undo 'red' if you mark the issue as resolved, methinks. If I raised maybe they want me to clear it.

Nice PR otherwise, thanks for all the work.

**Review by eddelbuettel — APPROVED** [review]

**Comment by dmurdoch** [comment]
Thanks!

**Comment by eddelbuettel** [comment]
I got distracted waiting for the CI and then forgot about the pending PR -- thanks again.  The versioned check is perfect. 

I am about to commit a simple fix (to create a default options array object) so that calling the viewer will work.

**Comment by dmurdoch** [comment]
The newer version of rgl is now on CRAN, so if you prefer a version requirement in DESCRIPTION that would work now, or you could leave in the run-time check and let people keep working with older versions.

**Comment by eddelbuettel** [comment]
Thanks, I saw via CRANberries so  congrats on a smooth transition.  I think the run-time check should do.

**Comment by eddelbuettel** [comment]
I got a little tied up with various other package updates (and still have some to do...) but I reckoned `rgl`, long updated at CRAN, is fine without a `RQuantLib` update as you rejigged the example to work with old and new `rgl`, correct?

**Comment by dmurdoch** [comment]
I think the RQuantlib changes should work fine with what's on CRAN now, and should be fine after the next update too. The most recent update on CRAN doesn't do the deprecation, the next one will.  I've just finished going through the revdeps on CRAN to let authors know they'll get deprecation warnings as of the next release.  I'm not sure when that will happen, but it'll likely be at least a week, more likely two.

**Comment by eddelbuettel** [comment]
Ah, that explains it -- I wasn't looking closely enough.

Especially once I have my "big fish" out of the way (`Rcpp` has been uploaded and I am waiting for 1 1/2 days now, `RcppArmadillo` and then `BH` are next) it would be a quick turnaround for this package.  Just let me if/when you need an update.

**Pull request merged**
